operator/builtin/input/azure/loganalytics/log_analytics.go

```diff
@@ -0,0 +1,113 @@
+package loganalytics
+
+import (
+	"context"
+	"fmt"
+
+	jsoniter "github.com/json-iterator/go"
+	"github.com/observiq/stanza/operator"
+	"github.com/observiq/stanza/operator/builtin/input/azure"
+	"github.com/observiq/stanza/operator/helper"
+)
+
+const operatorName = "azure_log_analytics_input"
+
+func init() {
+	operator.Register(operatorName, func() operator.Builder { return NewLogAnalyticsConfig("") })
+}
+
+// NewLogAnalyticsConfig creates a new Azure Log Analytics input config with default values
+func NewLogAnalyticsConfig(operatorID string) *LogAnalyticsInputConfig {
+	return &LogAnalyticsInputConfig{
+		InputConfig:   helper.NewInputConfig(operatorID, operatorName),
+		PrefetchCount: 1000,
+		StartAt:       "end",
+	}
+}
+
+// LogAnalyticsInputConfig is the configuration of a Azure Log Analytics input operator.
+type LogAnalyticsInputConfig struct {
+	helper.InputConfig `yaml:",inline"`
+
+	// required
+	Namespace        string `json:"namespace,omitempty"         yaml:"namespace,omitempty"`
+	Name             string `json:"name,omitempty"              yaml:"name,omitempty"`
+	Group            string `json:"group,omitempty"             yaml:"group,omitempty"`
+	ConnectionString string `json:"connection_string,omitempty" yaml:"connection_string,omitempty"`
+
+	// optional
+	PrefetchCount uint32 `json:"prefetch_count,omitempty" yaml:"prefetch_count,omitempty"`
+	StartAt       string `json:"start_at,omitempty"       yaml:"start_at,omitempty"`
```

Comment:
Is there any difference between the configuration values between the two azure inputs? If not, we should consider pulling them into a new type and embedding that type here. (Same idea as is done with `helper.InputConfig`). This can also ensure that validation is consistent between the two.

Reply:
```go
type LogAnalyticsInputConfig struct {
	helper.InputConfig `yaml:",inline"`
	AzureConfig        `yaml:",inline"`
}
```